### PR TITLE
Implement Sign-Up View

### PR DESCRIPTION
**Pull request by croberts808**
## What Changed:
In signup.ejs, I added the HTML so that a user can input name, username, and email then click "Create Account." 

Also, you can now click the HeelMeals title or hamburger icon and be routed back to the homepage. 

## How it Looks:
Screenshot (606)

## What _isn't_ Working
For some reason, when I click "Create Account" it routes to the login page. It's late and I'm sick so I'm probably missing something obvious here, feel free to add review comments if you see what's wrong and I'll change that before it's merged in!

### CODE REVIEW

File: frontend/signup.ejs [hunk 1]
```diff
@@ -12,4 +12,32 @@
         lunch_dining
         </span>
     <h1 style="margin-left: 5px; margin-top: 0px">HeelMeals</h1>
-</div>
\ No newline at end of file
+</div>
+
+<form action="/login" method="POST">
```
Comment (baileymeredith) [inline frontend/signup.ejs]: The 'Create Account' button is routing to the login page due to this line. If you change this to action='/user/new/' and keep the method="POST", then it'll be calling the correct route in server.js! This route will then create the user, add it to the db, and then redirect the user to a new view page!

**Review by baileymeredith — CHANGES_REQUESTED** [review]
First, this looks so great Catherine I'm so excited!!! Also, I think I found the problem and added a comment pointing to that line in signup.ejs.

File: frontend/signup.ejs (hunk 1 above)
Comment (croberts808) [reply frontend/signup.ejs]: I see! Thank you for pointing that out. I've corrected that here.

**Review by baileymeredith — APPROVED** [review]
LGTM! Thanks Catherine!

**Pull request merged**